requirements.txt

```diff
@@ -2,4 +2,4 @@ pycodestyle>=2.10
 pylint>=2.17
 coverage>=7.2
 sphinx>=7.0
-pyvcg[api]==1.0.6
```

Comment:
why was it necessary for you to remove the extra [api]?

Reply:
rules_python does not support right now this semantics.

Reply:
I found a different solution